keybert/_model.py

```diff
@@ -175,6 +182,11 @@ def _extract_keywords_single_doc(self,
             doc_embedding = self.model.embed([doc])
             candidate_embeddings = self.model.embed(candidates)
 
+            # Guided KeyBERT with seed keywords
+            if seed_keywords is not None:
+                seed_embeddings = self.model.embed([" ".join(seed_keywords)])
+                doc_embedding = np.average([doc_embedding, seed_embeddings], axis=0, weights=[3, 1])
```

Comment:
Why are we giving 3 times more weight to `doc_embedding`? Where does the number 3 come from?

Reply:
We compute a weighted average to make sure that the guided keywords only nudge the keyword extraction. For example, if we were to give the seeded keywords more weight than the actual document, we were to overfit the seeded keywords. 

The actual value of 3 is merely through experimentation but will most likely be a separate hyperparameter that the user can use. However, thus far 3 seems to be a nice balance between keeping the document embedding as the main entity to compare with the seeded keywords nudging (and not pushing) the keyword extraction.